### PR TITLE
Stacksmith merger

### PR DESCRIPTION
**Pull request by sameersbn**
- uses `gcr.io/stacksmith-images/ubuntu-buildpack:14.04-r06`
- uses ruby harpoon module
- removes `BITNAMI_APP_USER` instruction

### CODE REVIEW

**Comment by sameersbn** [comment]
Ready for merge. Tested a redmine/mariadb deployment with the updated image
/cc @prydonius

File: README.md [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
-> It has an elegant syntax that is natural to read and easy to write.
+> Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
 
 [ruby-lang.org](https://www.ruby-lang.org/en/)
 
 # TLDR
 
 ```bash
-docker run -it --name ruby bitnami/ruby
+docker run -it --name ruby -v /path/to/app:/app bitnami/ruby:latest ruby script.rb
```
Comment (prydonius) [inline README.md]: Given there is already a section on running a script in the documentation, I think the TLDR command should just start `irb`.

File: README.md [hunk 2]
```diff
@@ -272,45 +239,38 @@ docker-compose start ruby
...
 
 ```
 bats test.sh
 ```
 
-# Changelog
+# Notable Changes
+
+## 2.3.1-r0 (2016-05-11)
+- Commands are now executed as the `root` user. If you wish to use some other user, please specify the `--user` argument to specify the username on the docker command line or specify the command using `sudo`
```
Comment (prydonius) [inline README.md]: We should probably mention user namespaces and link to relevant docs?

File: README.md (hunk 1 above)
Comment (sameersbn) [inline README.md]: My intention was to have identical setup in the docker cli and docker compose examples of the TLDR section.

File: README.md (hunk 2 above)
Comment (sameersbn) [inline README.md]: linked to https://docs.docker.com/engine/reference/commandline/daemon/#daemon-user-namespace-options

File: README.md (hunk 1 above)
Comment (sameersbn) [inline README.md]: Let me know if you want me to change it back.

Comment (prydonius) [inline README.md]: Fair point, the use cases of Docker CLI and Docker Compose are fairly different though, so I think it's fine for their respective TLDRs to be different too

File: README.md [hunk 3]
```diff
@@ -1,40 +1,36 @@
...
 ## Docker Compose
 
 ```
 ruby:
-  image: bitnami/ruby
+  image: bitnami/ruby:latest
   command: ruby script.rb
   volumes:
-    - /path/to/ruby/app:/app
+    - /path/to/app:/app
```
Comment (prydonius) [inline README.md]: should this be relative?

Comment (sameersbn) [inline README.md]: can change it to be relative. maybe we can use `./code:/app`?

Comment (sameersbn) [inline README.md]: p.s. cannot use relative paths in docker-cli.

Comment (prydonius) [inline README.md]: I think `.:/app` is more likely

File: README.md (hunk 1 above)
Comment (sameersbn) [inline README.md]: this has be restored.

File: README.md (hunk 3 above)
Comment (sameersbn) [inline README.md]: hmm.. ok

**Comment by sameersbn** [comment]
updated with the `irb` sample and code path changes in docker-compose examples

**Comment by prydonius** [comment]
sweet!

**Pull request merged**